data_model/1.1/clusters/ClientMonitoring.xml

```diff
@@ -57,126 +57,112 @@ Davis, CA 95616, USA
 
 :xrefstyle: basic
 -->
-<cluster xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd cluster cluster.xsd" name="Mode Base Cluster" revision="1">
+<cluster xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd cluster cluster.xsd" name="Client Monitoring Cluster" revision="1">
```

Comment:
This cluster was removed from the spec at some point, and I THINK it was always meant to be provisional, but 1.1 didn't have provisional markers in the clusters themselves so none of that information will appear in the DM files.

Can either update the adoc in the branch or do a fixup in the script. I've done a few script fixups for 1.2, but 1.1 will be more extensive. If you check the spec itself it might have the provisional stuff listed out properly somewhere. But we were definitely a bit lax about this in the beginning.